pkg/common/common.go

```diff
@@ -100,12 +100,8 @@ const (
 	UnsupportedNodeAgentServerArgsAnnotation = "oadp.openshift.io/unsupported-node-agent-server-args"
 )
 
-const defaultMode = int32(420)
-
-func DefaultModePtr() *int32 {
-	var mode int32 = defaultMode
-	return &mode
-}
+// Owner can read/write; Group and Public do not have any permissions
+const DefaultPermission = int32(0600)
```

Comment:
Thinking about this, `600` means the owner can read and write the secret. Generally secrets are ment to be read-only, typically we do not modify secrets. I am thinking we can be more restrictive here, What about 400 or 440 ? just make the secret read-only.

Reply:
I think is ok

This is applied only to Secret data inside Pod, right? User could still see Secret contents looking Secret object in cluster